src/index.ts

```diff
@@ -189,6 +192,7 @@ async function processFile(
     core.setOutput('team', team);
     core.setOutput('is-migration', isMigration);
     core.setOutput('target-env', env);
+    core.setOutput('relevant-commits', JSON.stringify(relevantCommits));
```

Comment:
You're running this once per file but I think you want to aggregate this across all the files and make one big thing that explains all the different promotions happening in this PR.